src/benchmark/operators/join_benchmark.cpp

```diff
@@ -46,7 +46,7 @@ std::shared_ptr<TableWrapper> generate_table(const size_t number_of_rows) {
   const auto chunk_count = table->chunk_count();
   for (ChunkID chunk_id{0}; chunk_id < chunk_count; ++chunk_id) {
     const auto chunk = table->get_chunk(chunk_id);
-    if (!chunk) continue;
+    Assert(chunk, "Unexpected nullpointer-chunk.");
```

Comment:
Suggestion for the text: `Did not expect deleted chunks here.` Maybe add a comment `// see (wherever we explain the nullptr thing)` if it fits into the same line?